ledger/src/blockstore.rs

```diff
@@ -542,12 +542,6 @@ impl Blockstore {
         let candidate_erasure_set = ErasureSetId::new(slot, candidate_fec_set_index);
         let candidate_erasure_meta: ErasureMeta = deserialize(candidate_erasure_meta.as_ref())?;
 
-        // Add this candidate to erasure metas to avoid blockstore lookup in future
```

Comment:
no reason to do this as it is only called at the end of `do_insert_shreds` which does not need to perform any lookups after.